corehq/apps/cachehq/cachemodels.py

```diff
@@ -110,11 +110,3 @@ class UserReportsDataSourceCache(GenerationCache):
     views = [
         'userreports/data_sources_by_build_info',
     ]
-
-
-class UserReportsReportConfigCache(GenerationCache):
```

Comment:
hey, so it's not "one of those two mixins" it's just `QuickCachedDocumentMixin` because otherwise you get the view caching which isn't properly invalidated anymore.

This means that we can't cache view access, which means that for the time being every load of the reports page is going to hit the DB for all UCRs which kind of sucks. Especially when all we got form the thing that was working great was removing a view from a database that has ~0 write throughput.

IMO this change as it stands significantly hurts performance instead of helps. :(